src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ManagedProjectSystemPackage.cs

```diff
@@ -22,7 +22,7 @@ namespace Microsoft.VisualStudio.Packaging
         termNames: new[] { "dotnetcore" },
         termValues: new[] { "SolutionHasProjectCapability:.NET & CPS" }
         )]
-    [ProvideMenuResource("Menus.ctmenu", 4)]
+    [ProvideMenuResource("Menus.ctmenu", 6)]
```

Comment:
>6 [](start = 41, length = 1)

curious, why isn't this just incremented to 5?

Reply:
Just me showing how many times it took me to get right. :) 

I'll know it back to 5.